File: web3rpc/rpc-specs/code-samples/python/kaia/configuration/forkStatus.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

forkNumber = 20

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.fork_status(forkNumber)

print(kaia_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/configuration/forkStatus.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const forkNumber = 20;

  provider.klay
    .forkStatus(forkNumber, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
